### PR TITLE
Add an option to change the compression level of backups

### PR DESCRIPTION
**Pull request by PotatoMaaan**
Hi, this adds an option to the config file to change the compression level of backups.
I made this so I can make use of ZFS deduplication which only works on uncompressed files.

### CODE REVIEW

File: server/filesystem/archive.go [hunk 1]
```diff
@@ -62,8 +62,21 @@ func (a *Archive) Create(dst string) error {
...
+		cl = pgzip.NoCompression
+	case "best_speed":
+		cl = pgzip.BestSpeed
+	case "best_compression":
+		cl = pgzip.BestCompression
+	}
+
 	// Create a new gzip writer around the file.
-	gw, _ := pgzip.NewWriterLevel(writer, pgzip.BestSpeed)
+	gw, _ := pgzip.NewWriterLevel(writer, cl)
```
Comment (parkervcp) [inline server/filesystem/archive.go:79]: Why would you default to no compression? I get that it's what you want the majority of users probably aren't going to want that.

Comment (PotatoMaaan) [reply server/filesystem/archive.go:79]: I might be missing something here but cl should be BestSpeed unless it's manually changed.
I've tested this with a config file that doesn't have the "compression_level" set and the backups this created had a 22.7% compression ratio, so it did compress them. After starting my modified version of wings the config file did also have the "best_speed" option set. 

When I manually changed the compression_level to "none" the files were much larger and had a compression ratio of 0%.

Comment (parkervcp) [reply server/filesystem/archive.go:79]: I misread the case statement.

I still think this is a very niche issue.

Comment (PotatoMaaan) [reply server/filesystem/archive.go:79]: Guess I can't argue with that. Although some people might also benefit from being able to select a higher compression level.

**Pull request merged**